mtucker01752-boston.md

```diff
@@ -0,0 +1,7 @@
+
+## Boston, MA
+
+### Places to eat
+- Tavern in the Square - http://taverninthesquare.com
```

Comment:
@vicmaster - great idea! I edit the file locally, now to figure out how to make github recognize the changes....